src/extension.ts

```diff
@@ -43,6 +43,8 @@ export function activate(context: vscode.ExtensionContext) {
 
 			for (let i = 0; i < vscode.workspace.workspaceFolders.length; i++) {
 				let workspaceFolder = vscode.workspace.workspaceFolders[i].uri.path;
+				workspaceFolder += workspaceFolder.slice(-1) === '/' ? '' : '/' //Ensure that there is a trailing slash on Macs. Should probably pull into a separate method and test
```

Comment:
Codacy has a fix for the issue: [Missing semicolon.](https://app.codacy.com/gh/codacy-acme/vscode-codacy-cli-integration/pullRequest?prid=8496386)
```suggestion
				workspaceFolder += workspaceFolder.slice(-1) === '/' ? '' : '/'; //Ensure that there is a trailing slash on Macs. Should probably pull into a separate method and test
```